### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by Montsant19**

### CODE REVIEW

File: packages/node-cli/README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ suitecloud account:setup
 
 ## Release Notes & Documentation
 To read the 2021.1 NetSuite's release notes and documentation, check the following sections of NetSuite's Help Center:
-- Read the latest updates under SuiteCloud SDK in the [Weekly Update section](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_158955980607.html).
+- Read the latest updates under SuiteCloud SDK in the [Help Center Weekly Updates](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_3798389663.html).
```
Comment (davidecorreu) [inline packages/node-cli/README.md:92]: Are we changing the root part of the link on purpose?
from (1): `https://system.netsuite.com/app/help/helpcenter.nl?fid=`
to (2): `https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=`
The other links have (2).

Comment (Montsant19) [reply packages/node-cli/README.md]: Fixed, @davidecorreu, you were right.

Comment (davidecorreu) [reply packages/node-cli/README.md]: 👍

**Comment by davidecorreu** [comment]
Wait until vscode bold headers and logo issue is solved to merge this story.

**Comment by alii-gator** [comment]
@Montsant19 I think you forgot to update the logo for unit-testing readme

**Review by davidecorreu — APPROVED** [review]

**Review by tourajvaziri — APPROVED** [review]

**Pull request merged**